CHANGES.md

```diff
@@ -1,6 +1,9 @@
 Unreleased
 ----------
 
+- Optimize `Ansi_color.strip` by copying substrings without ANSI
```

Comment:
CHANGES for user facing stuff. I don't think the common user knows enough about dune's internals to care enough about this change.